system/class/Template.php

```diff
@@ -508,7 +508,7 @@ static function breadcrumbs($breadcrumbs = array())
 
         // zjistit aktivni stranku a jeji uroven
         list($pageId, $pageData) = PageManager::getActive();
-        if ($pageData !== null) {
+        if (is_array($pageData)) {
```

Comment:
I'd rather fix `PageManager::getActive()` to return `null` data as it should. It should probably check `$_index['is_page'] && $_index['is_found']`.

Reply:
Should I fix it this way or do you want to write it on TODO?  😁

Reply:
You can do it of course, should be a small change.